rotkehlchen/chain/manager.py

```diff
@@ -1710,10 +1725,26 @@ def get_eth2_validators(self) -> List[Eth2Validator]:
             raise ModuleInactive('Cant get eth2 validators since the eth2 module is not active')
         return DBEth2(self.database).get_validators()
 
+    def edit_eth2_validator(self, validator_index: int, ownership_proportion: FVal) -> None:
+        """Edit a validator to modify its ownership proportion. May raise:
+        - ModuleInactive if eht2 module is not active
+        - InputError if no row was affected
+        """
+        eth2 = self.get_module('eth2')
+        if eth2 is None:
+            raise ModuleInactive('Cant edit eth2 validators since the eth2 module is not active')
+        changes = DBEth2(self.database).edit_validator(
+            validator_index=validator_index,
+            ownership_proportion=ownership_proportion,
+        )
+        if changes != 1:
+            raise InputError(f'{changes} changes were made in the database.')
```

Comment:
This error makes no sense, especially to a user who would see it. Tells them nothing about what went wrong or where. Just do what I said inside the `edit_validator()` function.

```suggestion
```